apps/mobile-wallet/ios/Podfile.lock

```diff
@@ -2164,7 +2164,7 @@ SPEC CHECKSUMS:
   ReactCommon: f7da14a8827b72704169a48c929bcde802698361
   RNCAsyncStorage: 826b603ae9c0f88b5ac4e956801f755109fa4d5c
   RNFlashList: b521ebdd7f9352673817f1d98e8bdc0c8cf8545b
-  RNGestureHandler: 2282cfbcf86c360d29f44ace393203afd5c6cff7
+  RNGestureHandler: 6fee3422fd8c81c5ee756fa72e3d1780e9943d9d
   RNLocalize: b77875884750cb6a58cd6865863fe2ba2729b72b
```

Comment:
When something is copied to clipboard, Android renders a "paste" button, but it doesn't work well

<img width="350" alt="image" src="https://github.com/user-attachments/assets/e57d4669-d36c-4707-bfea-0a37ab4e3a53">

Reply:
Fixed

Reply:
styles look good!

I just checked it and it gives me an error when I click "Paste" and then "Continue"

<img width="350" alt="image" src="https://github.com/user-attachments/assets/c9e978da-ef79-4961-ad36-f12f4dea5848">

Reply:
Fixed in:
- https://github.com/alephium/alephium-frontend/pull/1033

Reply:
Thanks a lot! Merged and rebased!